pkg/api/v1/helper/helpers_test.go

```diff
@@ -26,6 +27,64 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 )
 
+func TestIsOpaqueIntResourceName(t *testing.T) { // inputs with the correct ResourceName prefix should pass
+	testCases := []struct {
+		resourceName v1.ResourceName
+		expectVal    bool
+	}{
+		{
+			resourceName: "pod.alpha.kubernetes.io/opaque-int-resource-foo",
```

Comment:
I still don't know what the correct prefix is;) `pod.alpha.kubernetes.io/opaque` or just `pod.alpha.kubernetes.io` or sth else.